### PR TITLE
Pbw 6610

### PR DESCRIPTION
**Pull request by aeng7**
-fixed an issue where adsManager.start was not called immediately after adsManager.init
-changed flow of certain paths since adsManager.start is called immediately
-removed unneeded unit tests

### CODE REVIEW

File: js/google_ima.js [hunk 1]
```diff
@@ -622,52 +621,18 @@ require("../html5-common/js/utils/utils.js");
...
-            this.adPlaybackStarted = true;
+            //resumeAd will only be called if we have exited fullscreen
+            //so this is safe to call
+            this.sharedVideoElement.webkitEnterFullscreen();
+            this.sharedVideoElement.play();
           }
-        }
-      };
-
-      this.requestPause = function()
```
Comment (pilievOoyala) [inline js/google_ima.js]: Do we not need requestPause and requestPlay anymore?

File: js/google_ima.js [hunk 2]
```diff
@@ -1582,6 +1547,7 @@ require("../html5-common/js/utils/utils.js");
           case eventType.USER_CLOSE:
           case eventType.SKIPPED:
           case eventType.COMPLETE:
+            this.adPlaybackStarted = false;
```
Comment (pilievOoyala) [inline js/google_ima.js]: I think this needs to get added to the resetvars()

File: js/google_ima.js (hunk 1 above)
Comment (aeng7) [inline js/google_ima.js]: Nope, the video wrapper is calling pauseAd and resumeAd directly now.

File: js/google_ima.js (hunk 2 above)
Comment (aeng7) [inline js/google_ima.js]: It is already in resetVars.

**Pull request merged**